src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,214 @@
+     */
+    @Prop({ reflect: true })
+    public value: number | string;
+
+    /**
+     * Name of icon for the info tile.
+     */
+    @Prop()
+    public icon?: string;
+
+    /**
+     * The text to show below the info tile. Long labels will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label?: string = null;
+
+    /**
+     * A string of text that is visually placed before the value.
+     */
+    @Prop({ reflect: true })
+    public prefix?: string;
+
+    /**
+     * A string of text that is visually placed after the value.
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * Set to `true` if info tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled? = false;
+
+    /**
+     * If supplied, the info tile will display a notification badge.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    /**
+     * Set to `true` to put the component in the `loading` state.
+     * This does _not_ disable the link. To do so, the
+     * `disabled` property should be set to `true` as well.
+     */
+    @Prop({ reflect: true })
+    public loading? = false;
+
+    /**
+     * If supplied, the info tile will be a clickable link.
+     */
+    @Prop()
+    public link?: Link;
+
+    /**
+     * Properties of the optional circular progress bar.
+     *
+     * Defaults:
+     * - `maxValue`: 100
+     * - `suffix`: %
+     * - `percentageColors`: false
+     *
+     * Colors change with intervals of 10 %.
+     */
+    @Prop()
+    public progress?: InfoTileProgress;
+
+    public render() {
+        const extendedAriaLabel =
+            this.checkProps(this?.prefix) +
+            this.value +
+            ' ' +
+            this.checkProps(this?.suffix) +
+            this.checkProps(this?.label) +
+            '. ' +
+            this.checkProps(this?.progress?.prefix) +
+            this.checkProps(this?.progress?.value) +
+            this.checkProps(this?.progress?.suffix) +
+            this.checkProps(this?.link?.title);
+
+        const link = !this.disabled ? this.link?.href : '#';
+
+        return [
+            <a
+                title={this.link?.title}
+                href={link}
+                target={this.link?.target}
+                tabindex="0"
+                aria-label={extendedAriaLabel}
+                aria-disabled={this.disabled}
+                class={{
+                    'is-clickable': !!this.link?.href && !this.disabled,
+                    'has-circular-progress':
+                        !!this.progress?.value || this.progress?.value === 0,
+                }}
+            >
+                {this.renderIcon()}
+                {this.renderProgress()}
+                <div class="value-group">
+                    {this.renderPrefix()}
+                    <div class="value-suffix">
+                        {this.renderValue()}
+                        {this.renderSuffix()}
```

Comment:
It's a little confusing that the div has a class `value-suffix`, but seems to hold both the value and the suffix?

Reply:
perhaps the class name should be called `value-and-suffix`

Reply:
fixed